rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-builder-6.5.0-release-notes.adoc

```diff
@@ -4,11 +4,30 @@ RPA Builder is the on-premises application that enables you to build automations
 
 include::partial$release-notes/rn-last-version-message.adoc[] 
 
-== TBD, 2023
+== November 28, 2023
 
 // == What's New
 include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
 
+* The new Signature Detection with AWS action step enables you to extract one or more signatures from a document. 
```

Comment:
You might consider combining these for better readability something like:
Several new action steps are available:
  * Signature Detection enables you to ...
  * Check Folder ...